features.bzl

```diff
@@ -16,6 +16,9 @@ _cc = struct(
     objc_linking_info_migrated = ge("7.0.0-pre.20230724.1"),
     # https://github.com/bazelbuild/bazel/commit/c8c3878088cb706b820d506a682e1156b7e8c64d
     swift_fragment_removed = ge("8.0.0-pre.20240101.1"),
+    # Whether the treat_warnings_as_errors feature works on macOS.
+    # https://github.com/bazelbuild/bazel/commit/3d7c5ae47e2a02ccd81eb8024f22d56ae7811c9b
+    treat_warnings_as_errors_works_on_macos = ge("7.1.0rc1"),
```

Comment:
Just FYI, release candidate levels are ignored when comparing versions, so this should be equivalent to `"7.1.0"`.